docs/reference/scientific-workflows.md

```diff
@@ -0,0 +1,43 @@
+
+A scientific workflow is a series of interconnected tasks or computational steps 
+that are designed to solve a specific problem or address a scientific question. 
+
+[Nextflow](https://www.nextflow.io/) and 
+[Cromwell](https://cromwell.readthedocs.io/en/stable/) are examples of tools that allow 
+researchers to define, automate, and execute such workflows.
+
+Scientific workflow can be beneficial when an analysis involves multiple steps, is 
+complex, needs to be repeated, involves different platforms or environments, or requires
+collaboration.
+
+## Workflows in Geneweaver
+Geneweaver uses scientific workflows to define analysis tools. This allows researchers
+to define complex analysis tools that can be used by other researchers in the Geneweaver
+web application.
+
+For single step analysis tools, the workflow definition can be inferred by the 
+Geneweaver web application. 
+
+### Why Use Workflows?
+Geneweaver uses scientific workflows as part of its analysis model for many of the 
```

Comment:
...for many of the same reasons scientists use them in their research - some of which are listed below."
could be added so that the reader knows you'll expand on what these reasons are.